MODULE.md

```diff
@@ -243,3 +251,65 @@ The Token Factory module as it is implemented on the Manifest Network, allows an
     - `exponent`: The exponent for the token e.g. how many zeros.
 
   **Example:** `manifestd tx tokenfactory modify-metadata utoken TOKEN "A token" 6`
+
+## COSM WASM Module
+
+The WASM module is responsible for handling the deployment and execution of WASM contracts on the Manifest Network.
+
+### Module Functionality
+
+The WASM module provides several key functionalities for managing smart contracts:
+
+#### Contract Deployment:
+
+- Upload WASM contract code to the blockchain
+- Instantiate contracts with initial parameters
+- Support for predictable contract addresses using instantiate2
+
+#### Contract Administration:
+
+- Set and clear contract administrators
+- Migrate contracts to new code versions
+- Update contract labels and configurations
+- Manage instantiation permissions
+
+#### Contract Execution:
+
+- Execute contract functions with custom messages
+- Support for contract-to-contract calls
+- Handle contract queries and responses
+
+#### Commands
+
+##### Upload Contract (upload-contract):
+
+- Syntax: `manifestd tx wasm store [wasm-file] [flags]`
+
+  - Parameters:
+    - `wasm-file`: Path to the .wasm contract file to upload
+    - Optional flags for instantiation permissions and access control
+
+  **Example:** `manifestd tx wasm store contract.wasm --from mykey`
+
+##### Instantiate Contract (instantiate-contract):
+
+- Syntax: `manifestd tx wasm instantiate [code-id] [json-encoded-args] --label [text] [flags]`
+
+  - Parameters:
+    - `code-id`: The ID of the uploaded contract code
+    - `json-encoded-args`: Init message for the contract in JSON format
+    - `label`: Human-readable label for the contract instance
+    - Optional flags for admin, funds, and other parameters
+
+  **Example:** `manifestd tx wasm instantiate 1 '{"count": 0}' --label "counter" --from mykey`
+
+##### Execute Contract (execute-contract):
+
+- Syntax: `manifestd tx wasm execute [contract-addr] [json-encoded-args] [flags]`
+
+  - Parameters:
+    - `contract-addr`: The address of the instantiated contract
+    - `json-encoded-args`: Execute message in JSON format
+    - Optional flags for sending funds with execution
+
+  **Example:** `manifestd tx wasm execute manifest14hj2tavq8fpesdwxxcu44rty3hh90vhujrvcmstl4zr3txmfvw9s0phg4d '{"increment": {}}' --from mykey`
```

Comment:
i am pretty sure the space will fail this, needs to be `'{"increment":{}}'`

Reply:
Fixed in c0d11ba805ea2c77c3dec3f91702f5f896439a63